plugin/pkg/admission/namespace/lifecycle/admission.go

```diff
@@ -45,8 +45,9 @@ const (
 	missingNamespaceWait = 50 * time.Millisecond
 )
 
-func init() {
-	admission.RegisterPlugin(PluginName, func(io.Reader) (admission.Interface, error) {
+// Register registers a plugin
+func Register(plugins *admission.Plugins) {
+	plugins.Register(PluginName, func(io.Reader) (admission.Interface, error) {
```

Comment:
@MHBauer yes, see https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/admission/plugin/namespace/lifecycle/admission.go#L54 for example